simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/configuration/SimulatorConfiguration.java

```diff
@@ -30,4 +31,27 @@
          */
         private int connectors;
     }
+
+    @Data
+    public static class WebSocketConfiguration {
```

Comment:
can you add also `@Builder` or `@Wither` annotation to simplify construction of this object?